### DISCUSSION TITLE
Alienvault integration

### DISCUSSION THREAD

**Discussion by CharRobin** [Security-Onion-Solutions/securityonion#7515] [Unsupported Versions]
Hi,

We have been looking at integrating the AlienVault into our instance of security onion and have come upon a error.

We have been following the documentation link below

https://docs.securityonion.net/en/2.3/alienvault-otx.html?highlight=otx

Once the script had ran it had shown a error and told me to navigate to a directory on the machine however the path that it told me to navigate to does not exist I have checked the hidden folders and cannot find it at all. Please can you help me with this issue.

Any help is greatly appreciated. 

Many Thanks,
Char

**Comment by dougburks** [comment]
What was the error?

What directory are you referring to?

**Comment by SanibelJack** [comment]
/opt/so/saltstack/default/pillar/zeek/init.sls
